modules/redhen_contact/redhen_contact.drush.inc

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Drush tools for the Redhen Contacts module.
```

Comment:
Seems like this file was probably an accidental include here? It has it's own patch on D.O I believe.

Reply:
Correct - I branched from 8.x-1.x and didn't realize these changes were included.
Looks like I pulled in 2 commits of yours by accident: f029132987283e749cb41adfa9c489a9c960e654 and 130a033d2c6e9a8a5908b4b47c8aac945e4a9bc8